frontend/src/app/patients/AddPatient.tsx

```diff
@@ -144,59 +147,61 @@ const AddPatient = () => {
   };
 
   return (
-    <main className={"prime-edit-patient prime-home"}>
-      <div className={"grid-container margin-bottom-4"}>
-        <PersonForm
-          patient={EMPTY_PERSON}
-          activeFacilityId={activeFacilityId}
-          savePerson={savePerson}
-          getHeader={(_, onSave, formChanged) => (
-            <div className="display-flex flex-justify">
-              <div>
+    <>
```

Comment:
Can you remove the fragment here so we don't have the diff?

Reply:
here and above, emm... sorry 🙈